NewTimesheet.py

```diff
@@ -0,0 +1,115 @@
+import requests
+import pandas
+import datetime
+from datetime import date
+import pprint
+import os
+
+
+# Set global variables from .env file
+V1_API_URL = "https://api.clockify.me/api/v1"
+WORKSPACE_ID = os.getenv("CLOCKIFY_WORKSPACE_ID")
+HEADERS = {"X-Api-Key": os.getenv("CLOCKIFY_API_KEY")}
+
+def responses_of_url():
+    '''Initial responses of url  '''
+    url = "{}/workspace/{}/users".format(V1_API_URL, WORKSPACE_ID)
+    responses = requests.get(url, headers=HEADERS).json()
+    return responses
+
+def gets_time_sunday():
+    ''' Get the last week range ''' 
```

Comment:
I think that you should change the docstring, like:
```suggestion
    ''' Return the last week sunday date''' 
```
You could check [this](https://www.python.org/dev/peps/pep-0008/#documentation-strings) to understand how to use docstring.